library/src/engine.cxx

```diff
@@ -290,29 +290,26 @@ engine::libInformation engine::getLibInfo()
   libInfo.BuildSystem = detail::LibBuildSystem;
   libInfo.Compiler = detail::LibCompiler;
 
-  std::string tmp;
-
 #if F3D_MODULE_RAYTRACING
-  tmp = "ON";
+  libInfo.Modules["Raytracing"] = true;
 #else
-  tmp = "OFF";
+  libInfo.Modules["Raytracing"] = false;
 #endif
-  libInfo.RaytracingModule = tmp;
 
 #if F3D_MODULE_EXTERNAL_RENDERING
-  tmp = "ON";
+  libInfo.Modules["ExternalRendering"] = true;
 #else
-  tmp = "OFF";
+  libInfo.Modules["ExternalRendering"] = false;
 #endif
-  libInfo.ExternalRenderingModule = tmp;
 
 #if F3D_MODULE_EXR
-  tmp = "ON";
+  libInfo.Modules["OpenEXR"] = true;
 #else
-  tmp = "OFF";
+  libInfo.Modules["OpenEXR"] = false;
 #endif
-  libInfo.OpenEXRModule = tmp;
 
+  // First version of VTK including the version check (and the feature used)
+#if VTK_VERSION_NUMBER >= VTK_VERSION_CHECK(9, 0, 20200527)
```

Comment:
how did that compiled before this change ?

Reply:
that's a mistake on my side, you removed that with the drop of VTK 9.0
fixed

Reply:
Ha right